IceCreamShop.playground/Contents.swift

```diff
@@ -71,8 +70,21 @@ class IceCreamShop {
 let lavenderHoney = Flavor(name: "Lavender Honey", rating: 5.0)
 let vanilla = Flavor(name: "Vanilla", rating: 4.5)
 let lavenderHoneyVanillaTwist = Flavor(name: "Lavender Honey Vanilla Twist", rating: 4.7)
-let iceCreams = [lavenderHoney, vanilla, lavenderHoneyVanillaTwist]
+var iceCreams = [lavenderHoney, vanilla, lavenderHoneyVanillaTwist]
 let prices: [Size] = [.small, .medium, .small]
 let toppings: [String] = ["Sprinkles", "Honey"]
 
 let viIceCreamShop = IceCreamShop(iceCreams: iceCreams, toppings: toppings, totalSales: 0.0)
+
+viIceCreamShop.listTopFlavors()
+
+let visLavenderHoneyCone = viIceCreamShop.orderCone(iceCream: lavenderHoneyVanillaTwist, toppings: ["Honey"], size: .large)
+visLavenderHoneyCone.eat()
+
+let eyveCone = viIceCreamShop.orderCone(iceCream: lavenderHoney, toppings: nil, size: .large)
```

Comment:
Try to use labels that are clear for most readers to understand what this value is.  An 'eyve Cone' is ambiguous.